script/deploy/facets/DeployExecutor.s.sol

```diff
@@ -32,6 +32,20 @@ contract DeployScript is DeployScriptBase {
 
         address erc20Proxy = json.readAddress(".ERC20Proxy");
 
-        return abi.encode(erc20Proxy);
+        // get path of global config file
+        string memory globalConfigPath = string.concat(
+            root,
+            "/config/global.json"
+        );
+
+        // read file into json variable
+        string memory globalConfigJson = vm.readFile(globalConfigPath);
+
+        // extract refundWallet address
+        address refundWalletAddress = globalConfigJson.readAddress(
+            ".refundWallet"
+        );
+
+        return abi.encode(erc20Proxy, refundWalletAddress);
```

Comment:
So the `refundWallet` is the `owner` of the contract correct? This address is able to withdraw any asset from `Executor` contract. Are there any additional roles or permissions assigned to the `refundWallet`, or is its access limited to functions like `withdrawToken` in all contracts that inherit from `WithdrawablePeriphery`?

Reply:
Yes and the only purpose it serves is to withdraw stuck funds